test/parallel/test-stringbytes-external.js

```diff
@@ -112,9 +112,11 @@ var PRE_3OF4_APEX = Math.ceil((EXTERN_APEX / 4) * 3) - RADIOS;
 (function() {
   // v8::String::kMaxLength defined in v8.h
   const kStringMaxLength = process.binding('buffer').kStringMaxLength;
+  const buf1 = new Buffer(kStringMaxLength + 1);
+  const buf2 = new Buffer(kStringMaxLength);
```

Comment:
This allocations needs to be after the `try catch` else it would fail on Raspberry Pi 1/2.